client/src/catalog-publish.html

```diff
@@ -60,7 +60,7 @@ <h1 class="page-title">Publish your element</h1>
 
         <input type="text" name="repo" value="{{repo::input}}" placeholder="user/repo">
         <div class="g-recaptcha" data-sitekey="6LcCRSgTAAAAAGhe5bUDS5lVHAPNN2cY7xo-Zk5f"></div>
-        <button on-tap="onSubmit">Publish</button>
+        <a href="/preview/[[repo]]"><button on-click="onSubmit">Publish</button></a>
```

Comment:
Using `<a>` is a bit weird here. You can right click to open in new window, tab order gets weird, etc. Can we just do this in `onSubmit`?